tests/test_resources/negative-scenarios/test_plugin_versions_collision/Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+
+stage('Read Evergreen YAML') {
```

Comment:
I think for this test case, the Jenkinsfile is not going to be executed, so maybe making this even simpler to avoid confusion?

Reply:
Since it's not going to be executed I've just copied an existing one. I don't think it's worth to invest time in this Jenkinsfile

Reply:
Up to you then, not something blocking for me.